File: Interfaz/ProyectoFinal/src/app/Components/Modulo/class/modulo.ts
export class Modulo{
    private idModulo!: number;
    private nombre!: string;
    private activo!: number;

    constructor(){}
    public get IdModulo(): number {
        return this.idModulo;
    }
    public set IdModulo(value: number) {
        this.idModulo = value;
    }
    public get Nombre(): string {
        return this.nombre;
    }
    public set Nombre(value: string) {
        this.nombre = value;
    }
    public get Activo(): number {
        return this.activo;
    }
    public set Activo(value: number) {
        this.activo = value;
    }
}

File: ProyectoFinal/ProyectoFinal/2_Dominio/Entidades/Modulo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal._2_Dominio.Entidades
{
    public class Modulo
    {
        public int IdModulo { get; set; }
        public string Nombre { get; set; }
        public bool? Activo { get; set; }
    }
}